src/test/unit/snykOss/services/vulnerabilityCount/ossVulnerabilityCountService.test.ts

```diff
@@ -0,0 +1,131 @@
+import { strictEqual } from 'assert';
+import sinon from 'sinon';
+import { IConfiguration } from '../../../../../snyk/common/configuration/configuration';
+import { TextDocument, TextEditor } from '../../../../../snyk/common/vscode/types';
+import { IVSCodeWindow } from '../../../../../snyk/common/vscode/window';
+import { IVSCodeWorkspace } from '../../../../../snyk/common/vscode/workspace';
+import { OssFileResult } from '../../../../../snyk/snykOss/ossResult';
+import { OssService } from '../../../../../snyk/snykOss/services/ossService';
+import { NpmModuleInfoFetchService } from '../../../../../snyk/snykOss/services/vulnerabilityCount/npmModuleInfoFetchService';
+import { OssVulnerabilityCountService } from '../../../../../snyk/snykOss/services/vulnerabilityCount/ossVulnerabilityCountService';
+import { ModuleVulnerabilityCountProvider } from '../../../../../snyk/snykOss/services/vulnerabilityCount/vulnerabilityCountProvider';
+import { LoggerMock } from '../../../mocks/logger.mock';
+
+suite('OSS VulnerabilityCountService', () => {
+  let workspace: IVSCodeWorkspace;
+  let window: IVSCodeWindow;
+  let config: IConfiguration;
+  let ossVulnerabilityCountService: OssVulnerabilityCountService;
+  let ossService: OssService;
+  let vulnerabilityCountProvider: ModuleVulnerabilityCountProvider;
+
+  setup(() => {
+    const logger = new LoggerMock();
+    const npmModuleInfoFetchService = {} as NpmModuleInfoFetchService;
+    ossService = {} as OssService;
+    workspace = {} as IVSCodeWorkspace;
+    window = {} as IVSCodeWindow;
+    config = {
+      isDevelopment: false,
+    } as IConfiguration;
+    vulnerabilityCountProvider = new ModuleVulnerabilityCountProvider(ossService, npmModuleInfoFetchService);
+
+    ossVulnerabilityCountService = new OssVulnerabilityCountService(
+      workspace,
+      window,
+      config,
+      vulnerabilityCountProvider,
+      ossService,
+      logger,
+    );
+  });
+
+  teardown(() => {
+    sinon.restore();
+  });
+
+  test("Doesn't activate in development", () => {
```

Comment:
please see comment in the file - if I remember correctly, it states that this should not activate in prod

Reply:
😅 It should be `Doesn't activate in production`, good catch 👍